test.sh

```diff
@@ -4,11 +4,17 @@ set -euo pipefail
 # switch into the repo root directory
 cd "$(dirname $0)"
 
+go get -t ./...
+
 echo -n "Running tests "
 function testrun {
     bash -c "umask 0; PATH=$PATH go test $@"
 }
 if [ ! -z "${COVERALLS:-""}" ]; then
+    go get golang.org/x/tools/cmd/cover
```

Comment:
Oops, none of this stuff was supposed to be in this PR. Can you remove all the changes to this file @mccv1r0 ?